app/views/layouts/_header.html.erb

```diff
@@ -15,6 +15,8 @@
   		<ul class="nav navbar-nav">
   			<% if user_signed_in? %>
 
+          <li> <%= link_to "Пригласить", new_invitation_path if current_user.admin? %> </li>
```

Comment:
If this link were use to invite teachers only, I'd rather place it in courses dropdown. Do we really need to invite students? Seems like they can sign up in a normal way.